jobs/appointment_reminder/requirements.txt

```diff
@@ -1,3 +1,7 @@
 Flask
 requests
 python-dotenv
+Flask-Mail
+Jinja2
+
+-e git+https://github.com/sumesh-aot/queue-management.git@email_job_fix#egg=api&subdirectory=api
```

Comment:
This needs to be changed, once this PR is merged and is in master branch, I can create another PR to point to bcgov repo